README.md

```diff
@@ -1,20 +1,94 @@
 ![](https://repository-images.githubusercontent.com/275870366/15880680-ba44-11ea-888b-8211b9431f94)
 
+
+
+
+
 <h1> HTML5Sortable </h1>
 
 [![Build Status](https://img.shields.io/travis/lukasoppermann/html5sortable/master.svg?style=flat-square)](https://travis-ci.org/lukasoppermann/html5sortable) [![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE.md) [![Coverage Status](https://img.shields.io/coveralls/lukasoppermann/html5sortable/master.svg?style=flat-square)](https://coveralls.io/github/lukasoppermann/html5sortable) [![Known Vulnerabilities](https://snyk.io/test/github/lukasoppermann/html5sortable/badge.svg?style=flat-square)](https://snyk.io/test/github/lukasoppermann/html5sortable)  [![NPM](https://img.shields.io/npm/v/html5sortable.svg?style=flat-square)](https://www.npmjs.com/package/html5sortable) [![npm](https://img.shields.io/npm/dt/html5sortable.svg?style=flat-square)](https://www.npmjs.com/package/html5sortable) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](CONTRIBUTING.md) [![Code of Conduct](https://img.shields.io/badge/code%20of-conduct-ff69b4.svg?style=flat-square)](CODE_OF_CONDUCT.md)
 
 > **Lightweight vanillajs micro-library for creating sortable lists and grids using native HTML5 drag and drop API.**
 
-## Community maintained
+
+
+# Table of Contents
+
+  * [Community maintained](#community-maintained)
+  * [Looking for Co-Maintainer](#looking-for-co-maintainer)
+  * [Features](#features)
+  * [Framework adapters](#framework-adapters)
+
+- [Installation](#installation)
+
+- [Examples](#examples)
+
+- [Docs](#docs)
+
+  * [Usage](#usage)
+
+  * [Styling](#styling)
+
+  * [Nesting](#nesting)
+
+  * [Events](#events)
+
+    + [sortstart](#sortstart)
```

Comment:
I feel we can drop the subitems. Events, Options, Methods, Benefits, Known Issues are enough, the subpoints don't need to be here. Otherwise the list gets too long.